### ISSUE TITLE
Cursor glitching on XQuartz, macOS 10.13.3

### ISSUE THREAD

**Issue by techfury90** [LM-3/usim#14]
usim appears to run fine other than some odd glitching that appears when the cursor is moved. I would hazard to guess that it may have something to do with the fact I was running it on a machine with DPI scaling. Who knows!

**Comment by techfury90** [comment]
Seems that the same effect occurs when you run usim from Ubuntu 17.10 amd64 displayed on the VcXsrv X server for Windows. Interestingly, on the exact same machine and X server, this problem does not occur while running usim in Ubuntu 16.04.3 for WSL...

Video of the symptoms: https://youtu.be/OvS-rD-0Yqc

The output is all like this:

tv: tv_read past end; offset 3330000
tv: tv_read past end; offset 3327740
tv: tv_read past end; offset 3330000
tv: tv_read past end; offset 3327740
tv: tv_read past end; offset 3113000
tv: tv_read past end; offset 3112740
tv: tv_read past end; offset 3113000
tv: tv_read past end; offset 3112740
tv: tv_read past end; offset 3047100
tv: tv_read past end; offset 3047040
tv: tv_read past end; offset 3047100
tv: tv_read past end; offset 3047040

**Comment by ThothXV** [comment]
Yeah, I'm encountering the same problem. It seems like this is caused by the DPI being high and the system thus being confused by why the mouse is moving so fast? but that's based on cursory observations. I'm not entirely sure.
